packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -522,6 +522,19 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         />
       );
       break;
+    case 'badRequest': {
+      const { title, moreDetails, disableStartCallButton, iconName } = getEndedCallPageProps(locale, endedCall);
+      pageElement = (
+        <NoticePage
+          iconName={iconName}
+          title={title}
+          moreDetails={callees ? '' : moreDetails}
+          dataUiId={'left-call-page'}
+          disableStartCallButton={disableStartCallButton}
```

Comment:
What does the end call page look like for the badRequest? presumably it should not have the rejoin button

Reply:
<img width="686" alt="Screenshot 2024-05-27 at 9 22 18 AM" src="https://github.com/Azure/communication-ui-library/assets/96077406/59c73ef8-fac1-4b07-bb3f-467edc32cc2c">